### PR TITLE
[wasm][debugger] Support "supportsDebuggerProperties" from DAP 

### PR DESCRIPTION
**Pull request by thaystg**
Implement support to dotnet.setDebuggerProperty from dap and give the Wasm programmer a better debugging experience.

For now we are only supporting JustMyCode flag, the others I think are not important to us, yet.

Depends on https://github.com/microsoft/vscode-js-debug/pull/1311

### CODE REVIEW

**Review by ilonatommy — APPROVED** [review]
Looks prefect

**Review by radical — APPROVED** [review]

**Comment by radical** [comment]
@thaystg @ilonatommy Should this, and https://github.com/dotnet/runtime/issues/43184 be moved to 8.0 milestone?

**Comment by ilonatommy** [comment]
We are blocked on approving https://github.com/microsoft/vscode-js-debug/pull/1311 before merging this one, so I guess we have no choice but move it.

**Review by lewing — APPROVED** [review]

**Pull request merged**
